python/hail/methods/impex.py

```diff
@@ -1064,6 +1087,21 @@ def import_matrix_table(paths, row_fields={}, key=[], entry_type=tint32, missing
 
     Notes
     -----
+
+    The structure of the imported matrix table is:
+
+        - **row fields** are named as specified in the column header. If they
```

Comment:
The interplay between `row_fields`, `key` and the actual row names in the file isn't clear from the docs. Does `row_fields` need to include all row names? Or do you drop those columns corresponding to row names that aren't in row_fields? I think `key` would be much clearer as `row_key`.